package.json

```
@@ -13,28 +13,28 @@
     "node": ">=16.18.0"
```

Was this intentionally left at 16? Is 16 really the oldest Node the published package works on?

---

No, this is an oversight. We should bump this to match `@types/node`.

```suggestion
    "node": ">=18.19.0"
```